sync/handler_blocks_response_test.go

```diff
@@ -122,27 +121,38 @@ func TestSyncing(t *testing.T) {
-	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // 11-20
-	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // 21-30
+	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // 1-11
+	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // 12-22
+	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // 23-23
 	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // NoMoreBlock
 
 	shouldPublishMessageWithThisType(t, networkAlice, message.MessageTypeBlocksRequest)
-	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // 31-40
-	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // 41-50
-	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // 51-60
+	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // 24-34
+	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // 35-45
+	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // 46-46
 	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // NoMoreBlock
 
 	shouldPublishMessageWithThisType(t, networkAlice, message.MessageTypeBlocksRequest)
-	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // 61-70
-	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // 71-80
-	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // 81-90
+	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // 47-57
+	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // 58-68
+	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // 69-69
+	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // NoMoreBlock
+
+	shouldPublishMessageWithThisType(t, networkAlice, message.MessageTypeBlocksRequest)
+	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // 70-80
+	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // 81-91
+	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // 92-92
 	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // NoMoreBlock
 
 	// Latest block requests
 	shouldPublishMessageWithThisType(t, networkAlice, message.MessageTypeBlocksRequest)
-	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // 91-100
+	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // 93-100
 	shouldPublishMessageWithThisType(t, networkBob, message.MessageTypeBlocksResponse) // Synced
```

Comment:
check this bundle to make sure it is synced

Reply:
Done